vision/cloud-client/product_search/product_in_product_set_management.py

```diff
@@ -21,14 +21,17 @@
 https://cloud.google.com/vision/product-search/docs/
 """
 
-# [START product_search_import]
 import argparse
 
+# [START vision_product_search_add_product_to_product_set]
+# [START vision_product_search_remove_product_from_product_set]

```

Comment:
these two tags only enclose the import line.  does not look right.

Reply:
These tags serve the purpose of including the import with the samples.